.release/linux/package/usr/lib/systemd/system/nomad.service

```diff
@@ -26,6 +26,7 @@ After=network-online.target
 # StartLimitInterval = 10s
 
 [Service]
+Type=notify
```

Comment:
Note to reviewers: ideally we'd have this as `Type=notify-reload` because that's a little nicer, but that's not available until systemd 253, which only the very most recent LTS distros have.